code/modules/antagonists/clock_cult/servant_of_ratvar.dm

```diff
@@ -60,6 +59,7 @@
 /datum/antagonist/servant_of_ratvar/apply_innate_effects(mob/living/M)
 	. = ..()
 	owner.current.faction |= "ratvar"
+	owner.language_holder.grant_language(/datum/language/ratvar, TRUE, TRUE, LANGUAGE_CULTIST)
```

Comment:
language_cultist is fine here and doesnt need something special since users cannot be a member of both cults